src/modeler/AutoRest.Swagger/Validation/AnonymousBodyParameter.cs

```diff
@@ -41,9 +41,12 @@ public class AnonymousBodyParameter : TypedRule<SwaggerParameter>
         public override ServiceDefinitionDocumentType ServiceDefinitionDocumentType => ServiceDefinitionDocumentType.ARM;
 
         /// <summary>
-        /// When to apply the validation rule, before or after it has been merged as a part of 
-        /// its merged document as specified in the corresponding '.md' file
-        /// By default consider all rules to be applied for After only
+        /// Whether the rule should be applied to the individual or composed context based on
+        /// the corresponding .md file
+        /// In most cases this should be composed
+        /// This is because validation rules that run in individual mode will end up
+        /// throwing multiple validation messages for the same violation if related model/property,etc 
+        /// was referenced in multiple files
```

Comment:
I wouldn't copy the whole explanation in a specific rule, a comment for the rule IMO, should justify why the value was picked.